.github/ISSUE_TEMPLATE/sensor_request.md

```diff
@@ -9,98 +9,36 @@ assignees: '@teresaho-biosensics'
-- [ ] Send Application RC build to operations team for V&V
-- [ ] App link to source code:
-
-**@biosensics/operations & @biosensics/quality :**
-- [ ] V&V Document Released
-- [ ] Complete DCO and submit to [Quality](mailto:quality@biosensics.com)
-- [ ] Hexnode portal/policy set up
-- [ ] Mobile device operating system is correct
-- [ ] Mobile device model is correct (version, iOS/Android, etc.)
-- [ ] Sensor firmware is correct
-- [ ] Device enrolled in Hexnode
-- [ ] Equipment set up for V&V execution
-- [ ] Parse account created in correct server
-- [ ] List parse account username:
-
-*Ensure all above items have been completed before continuing*
+***Pre-Manufacturing Checklist:***
 
 **@biosensics/test-and-manufacturing :**
-- [ ] V&V executed
-- [ ] Completed V&V submitted to [Quality](mailto:quality@biosensics.com)
-
-**@biosensics/quality :**
-- [ ] Approve completed V&V
+- [ ] Manufacturing Report Created for the Month
+- [ ] Welder Maintenence Completed for the Month [Date Completed: ]
+- [ ] Welder Logs for Previous Month Uploaded [Link: ]
+- [ ] Batch Number Updated in Spreadsheet [Updated Batch Number: ]
 
-**@biosensics/engineering :**
 
-Server:
-- [ ] Send tagged release source code zip to @biosensics/quality
-- [ ] Create server release
-- [ ] Deploy server release to Aptible production server
-- [ ] Confirm any updated Aptible production server configuration values
-- [ ] Merge `release` branch into `develop` on server repository
-
-Application:
-- [ ] Merge `release` branch into `master` on Application repository
-- [ ] Send release APK and installer to operations and quality for production use
+***Manufacturing Checklist:***
 
+**@biosensics/test-and-manufacturing :**
 
-**@biosensics/quality :**
-- [ ] App ECO closed & filed to QSR
-- [ ] Server ECO closed & filed to QSR
-- [ ] Insert file path(s) to documentation (ECO, V&V etc):
+|   	| Board ID 	| Manu Tested 	| Firmware Uploaded 	| Welded 	| Calibrated 	| Epoxied 	| Verified and Ready for Shipment 	|
+|---	|----------	|-------------	|-------------------	|--------	|------------	|---------	|---------------------------------	|
+| 1 	|          	|             	|                   	|        	|            	|         	|                                 	|
+| 2 	|          	|             	|                   	|        	|            	|         	|                                 	|
+| 3 	|          	|             	|                   	|        	|            	|         	|                                 	|
+| 4 	|          	|             	|                   	|        	|            	|         	|                                 	|
+| 5 	|          	|             	|                   	|        	|            	|         	|                                 	|
 
-*Ensure all above items have been completed before continuing*
-- [ ] New version approved for customer use
 
-**@biosensics/operations :**
-- [ ] Confirm telehealth meeting room on release build
```

Comment:
do you want to capture any summary information? Number of units manufactured? Number passed? Number failed? Item location?